client/src/components/Console.tsx

```diff
@@ -95,8 +91,18 @@ export default function Console() {
       fetchDocuments();
       setRefreshNeeded(false);
     }
-  }, [selectedView, refreshNeeded]);
+  }, [refreshNeeded]);
+  useMemo(() => {
+    const entireMunicipalityDocs = tmpDocuments.filter(
+      (doc) => doc.doc_coordinates?.type === "EntireMunicipality"
+    );
+    const otherDocs = tmpDocuments.filter(
+      (doc) => doc.doc_coordinates?.type !== "EntireMunicipality"
+    );
 
+    setEntireMunicipalityDocuments(entireMunicipalityDocs);
```

Comment:
https://github.com/user-attachments/assets/3e60eb20-a0c0-4ac5-a324-2a8bc504d3ea

after filtering the dates, the result does not contain related string, also crashes!